packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -141,6 +151,8 @@ class TestCommand extends FlutterCommand {
     }
 
     commandValidator();
+    final List<String> names = argResults['name'];
```

Comment:
Apparently it's null; didn't see any error when running tests without these flags.